hack/verify-examples.sh

```diff
@@ -74,8 +74,17 @@ if [[ "$1" == "windows" ]]; then
     )
 fi
 
+cleanup_examples() {
+    for EXAMPLE in "${EXAMPLES[@]}"; do
+        echo "Attempting deletion of \"$EXAMPLE\""
+        kubectl delete -f "$EXAMPLE" --ignore-not-found
+    done
+}
+
+trap cleanup_examples EXIT
+
 for EXAMPLE in "${EXAMPLES[@]}"; do
-    rollout_and_wait $EXAMPLE
+    rollout_and_wait "$EXAMPLE"
```

Comment:
Why is this change needed?

Reply:
If there's a space in $EXAMPLE it will split into multiple args. For example

```bash
#!/usr/bin/env bash

hello_world() {
  echo $1
}
VAR="Hello, world"
hello_world $VAR
hello_world "$VAR"
```
Prints:
```
Hello,
Hello, world!
```

I looked at the context, and it was clear that $EXAMPLE is meant to be a single argument, so I fixed it.

Reply:
Ah ok, thanks for explaining!